api/controller.py

```diff
@@ -308,30 +305,55 @@ def load_lane(self, language_key, name):
             )
         return lanes[name]
 
-    def load_licensepool(self, data_source, identifier_type, identifier):
-        """Turn user input into a LicensePool object."""
-        if isinstance(data_source, DataSource):
-            source = data_source
-        else:
-            source = DataSource.lookup(self._db, data_source)
-        if source is None:
-            return INVALID_INPUT.detailed(_("No such data source: %(data_source)s", data_source=data_source))
+    def load_work(self, library, identifier_type, identifier):
```

Comment:
I know why you do this, but technically load_work() shouldn't require a Library as an argument. Not sure if it's worth changing in this branch though.